pages/docs/custom-dashboards.mdx

```diff
@@ -0,0 +1,65 @@
+---
+description: Create self-service dashboards to visualize, monitor, and share insights from your LLM application data.
+---
+
+# Custom Dashboards
+
+<AvailabilityBanner
+  availability={{
+    hobby: "public-beta",
+    core: "public-beta",
+    pro: "public-beta",
+    enterprise: "public-beta",
+    selfHosted: "not-available",
```

Comment:
why not available self-hosted?

Reply:
@marcklingen I want to get feedback on cloud first to see if we break the data model right away. Limiting it here allows for a couple more configurations before we set it into stone.